orchestra/bots/assignment_policies.py

```diff
@@ -6,3 +6,4 @@ def staffbot_autoassign(task, **kwargs):
     request_cause = StaffBotRequest.RequestCause.AUTOSTAFF.value
     bot = StaffBot()
     bot.staff(task.id, request_cause=request_cause)
+    return task
```

Comment:
ok cool---whereas preassigning workers modifies the task, i don't think this function should.  returning the task suggests it might have been mutated.

that said i don't feel terribly strongly, so if you feel consistency is better, i can understand that.